apps/webapp/api/main.ts

```diff
@@ -32,13 +37,32 @@ const transformHeadline = (headline: string): Promise<string> => {
   });
 };
 
-v1Api.post("/headline", async (c: Context) => {
+v1Api.post("/api/headline", async (c: Context) => {
```

Comment:
By changing this to `/api/headline`, the curl request I need to use becomes:

```bash
➜  trust-assembly git:(deploy_anil_setup_serve_command) curl --request POST \
    --url localhost:8000/api/v1/api/headline \
    --json '{
                "headline": "blah"
        }'                                                                                                                                                                                     
{"transformedText":"BLAH"}%

```

note the redundant `/api/v1/api/headline`.

So I think we just want `v1Api.post("/headline", async (c: Context) => {`

Reply:
good catch. Added a fixup